### PR TITLE
 ✨ Adding Poisson tunning set for workload generation

### PR DESCRIPTION
**Pull request by dumb0002**
## Summary

This PR adds the Poisson tuningset in the workload configuration for clusterloader2 - we will use the tuning set to generate the KS benchmark workloads in our experiments. 

N.B: this PR also improves the README to allow users to easily copy & past the list commands. 

## Related issue(s)

Fixes #

### CODE REVIEW

File: test/performance/workloads/performance-test-config.yaml [hunk 1]
```diff
@@ -41,6 +41,12 @@ tuningSets:
   parallelismLimitedLoad:
     parallelismLimit: 100
 
+# PoissonLoad defines a load with with Poisson arrival time
```
Comment (ezrasilvera) [inline test/performance/workloads/performance-test-config.yaml:44]: Typo:  Remove one of the "with"

Comment (ezrasilvera) [inline test/performance/workloads/performance-test-config.yaml:44]: Arrival time of what ?  Maybe add ?

File: test/performance/workloads/performance-test-config.yaml [hunk 2]
```diff
@@ -41,6 +41,12 @@ tuningSets:
...
+# for a given rate parameter λ (ExpectedActionsPerSecond)
```
Comment (ezrasilvera) [inline test/performance/workloads/performance-test-config.yaml:45]: I would change the order 
for a given rate parameter ExpectedActionsPerSecond (λ in the Poisson model)

File: test/performance/workloads/performance-test-config.yaml (hunk 1 above)
Comment (dumb0002) [reply test/performance/workloads/performance-test-config.yaml:44]: Fixed! thanks

Comment (dumb0002) [reply test/performance/workloads/performance-test-config.yaml:44]: The Poisson arrival time is for the load itself.

File: test/performance/workloads/performance-test-config.yaml (hunk 2 above)
Comment (dumb0002) [reply test/performance/workloads/performance-test-config.yaml:45]: Fixed! Thanks

**Pull request merged**